products/amazon-documentdb.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Amazon DocumentDB
+category: service
+tags: amazon database
+iconSlug: amazondocumentdb
+permalink: /amazon-documentdb
+releasePolicyLink: https://docs.aws.amazon.com/documentdb/latest/developerguide/compatibility.html
+releaseDateColumn: true
+releaseColumn: true
+
```

Comment:
Do you think a custom column documenting the MongoDB compatibility is needed ?

An example of a custom column can be found on https://endoflife.date/apache-camel (the "Java support" column).

Reply:
I like this idea in general, but I think in this case, the DocumentDB version maps directly to the MongoDB version so it would feel a bit redundant. WDYT?